src/www/authgui.inc

```diff
@@ -409,6 +442,16 @@ function display_login_form($Login_Error)
           <label for="passwordfld"><?=gettext("Password:"); ?></label>
           <input id="passwordfld" type="password" name="passwordfld" class="form-control pwd" tabindex="2" />
         </div>
+<?php
+    // Determine if we need to display the MFA code field
+    $showMFAField = isMFARequired($config);
+
+    if ($showMFAField): ?>
+        <div class="form-group">
+            <label for="mfacode"><?= gettext("MFA Code:"); ?></label>
+            <input type="text" class="form-control" name="mfacode" id="mfacode" value="" />
```

Comment:
Are there cases where MFA wouldn't be a number? Otherwise we might want to improve browser hinting for the auto-completion (e.g., password manager) by adding `inputmode="numeric"`, `autocomplete="one-time-code"` as well as `pattern="\d*"`.